src/subsCache.coffee

```diff
@@ -1,6 +1,41 @@
 # debug = (args...) -> console.log.apply(console, args)
 debug = (args...) -> return
 
+makeDelegatingCallbackFn = (cachedSub, callbackName)->
```

Comment:
I'm not completely happy with it either. I want to convey that calling this function causes the registered callbacks to be called in the same way. That is, "you can pretend that this is the original callback".

Maybe "makeOnStopPassthroughFn" or something along those lines? I don't know.